pghoard/transfer.py

```diff
@@ -58,8 +64,23 @@ def form_key_path(file_to_transfer):
                             file_to_transfer["filetype"],
                             name)
 
+    def transmit_statsd_metrics(self):
+        """Keep statsd updated about how long time ago each filetype was successfully uploaded"""
+        for site in self.state:
+            for filetype, prop in self.state[site]["upload"].items():
+                if prop["last_success"]:
+                    self.stats.gauge(
+                        "pghoard.last_upload_age",
+                        time.time() - prop["last_success"],
+                        tags={
+                            "site": site,
+                            "type": filetype,
+                        }
+                    )
+
     def run(self):
         while self.running:
+            self.transmit_statsd_metrics()
```

Comment:
every thread transmits stats (from the shared dict?) once per second, isn't that a bit too much?